client/components/sectionform/sectionform.html

```diff
@@ -2,7 +2,7 @@
   <div class='col-xs-12'>
     <h4> Edit Section </h4>
 
-    <form class="form" name="form" ng-submit="submitForm(form)" novalidate ng-show="!success">
+    <form class="form" name="form" novalidate ng-show="!success">
```

Comment:
yes, the issue with that was the buttons caused the form submission

The issue is similar to this issue...
http://stackoverflow.com/questions/12319758/angularjs-clicking-a-button-within-a-form-causes-page-refresh

Hence i prefer the ng-click/ng-enter syntax, as i think the behavior is more regular. Changed it back anyway and added the necessary type='button'